content/blog/monthly-april-2020.md

```diff
@@ -0,0 +1,101 @@
+---
+title: This Month in TiKV - April 2020
+date: 2020-04-30
+author: TiKV Authors
+---
+
+April 1st marked TiKV’s 4th birthday. We are grateful for the support and contribution of the community while focusing on making better TiKV.
+
+Let’s take a look at what we have achieved in April!
+
+## News
+
+We're working on another API- VerKV, a multi-versioned KV between RawKV and TxnKV, to help TiKV expand application scenarios. The work plan proposed by [@fredchenbj](https://github.com/fredchenbj) is available [here](https://github.com/tikv/tikv/issues/7295) and join us by picking the task you are interested in.
```

Comment:
```suggestion
We're working on another API - VerKV, a multi-versioned KV without transactions, to help TiKV expand application scenarios. Its an API halfway between RawKV and TxnKV. The work plan proposed by [@fredchenbj](https://github.com/fredchenbj) is available [here](https://github.com/tikv/tikv/issues/7295); join us by picking a task you are interested in!
```

Could also add a link to the RFC: https://github.com/tikv/rfcs/blob/master/text/2020-03-11-versioned-kv.md